practices/unhappy_clients.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: unhappy-clients
+stages:
+  - development
+short_description: The article describes the way how to deal with unhappy clients. No matter how good you are at what you do, sooner or later you'll come across an unhappy client.  Bad feedback often leads to great work, as you learn valuable lessons and really push your skills.
+tags:
+  - Unhappy Clients
+  - communication
+  - feedback
+  - problem identification
+  - extra mile
+keywords:
+  - client
+  - communication
+  - complaint
+  - planning
+  - methodology
+---
+
+# Unhappy Clients
+
+## TL;DR
+
+No matter how good you are at what you do, sooner or later you'll come across an unhappy client. Although your ego can get hurt, don't be disheartened. Bad feedback often leads to great work, as you learn valuable lessons and really push your skills.
+
+So, what to do in such a situation?
+
+![](/files/keep_calm.png)
+
+[Keep Calm and Carry On *(thanks to Ministry of Information – UK, 1939)*](/files/keep_calm.png)
+
+## How to Deal With Unhappy Clients
+
+**1. Communicate in person**
+
+When dealing with complex issues – if possible – use one-to-one communication. Don't write your response; request a face-to-face meeting. At least, arrange a phone call.
+
+**2. Keep calm and carry on**
+
+It's natural to react negatively if you receive a complaint from a client. It is easily possible you feel upset and offended, which only makes you defensive and puts you in danger of responding to your client in a negative manner. In such a case, stay calm and take a step back from the situation.
+
+**3. Apologize respectfully**
+
+Approach the problem directly by apologizing to the client. This can help the client to calm down and also allow you both to talk professionally about next steps.
+
+**4. Identify the problem**
+
+Once everyone is feeling calm, ask the client why they are unhappy and what you can do to fix the situation. Don't accept vague responses such as "I just don't like it". Try and get specifics by asking many questions. Ask for examples of what they like and dislike about the work.
+
+**5. Re-establish the brief**
+
+Sometimes, it is wise to start from scratch and re-establish the project requirements. Make sure you know exactly what the client wants. Repeat things back to the client to make sure you've got it right – for example, "So, I understand that you want _xxx_. Is that right?".
+
+**6. Written word remains**
+
+Once you've found out what needs to be done, get it all in writing. This is a case of 'covering your own ass' a little, so you can refer back in future, should any issues arise. This can simply be drafted in an email, or via an 'official' document, outlining all the project objectives.
+
+**7. Go the extra mile**
+
```

Comment:
Would also mention the importance of MVP and Clickable prototype: https://developerexperience.io/practices/minimum-viable-product, https://developerexperience.io/practices/clickable-prototype. yet I am unsure where to put it.

Reply:
Amended